tests/phpunit/CRM/Contact/BAO/QueryTest.php

```diff
@@ -754,6 +754,39 @@ public function testRelationshipClause() {
     $this->assertLike($where2, $sql5[2]);
   }
 
+  public function testRelationshipDescription() {
+    $contactID_a = $this->individualCreate(1);
```

Comment:
@colemanw The parameter should be an array https://github.com/civicrm/civicrm-core/blob/master/Civi/Test/ContactTestTrait.php#L78